src/components/ec-timer/ec-timer.vue

```diff
@@ -74,22 +89,40 @@ const props = defineProps({
 });
 const emit = defineEmits(['time-expired']);
 
-const diameter = computed(() => radius.value * 2 + strokeWidth.value);
+const radius = 24;
+const strokeWidth = 4;
+const diameter = computed(() => radius * 2 + strokeWidth);
 const viewbox = computed(() => `0 0 ${diameter.value} ${diameter.value}`);
-const circumference = computed(() => 2 * Math.PI * radius.value);
+const circumference = computed(() => 2 * Math.PI * radius);
 const steps = computed(() => circumference.value / props.seconds);
-const offset = computed(() => circumference.value + steps.value * secondsLeft.value);
 
-const radius = ref(24);
-const strokeWidth = ref(4);
-const secondsLeft = ref(props.seconds);
+const offset = computed(() => {
+  if (props.isRunning) {
+    return circumference.value + steps.value * totalSecondsLeft.value;
+  }
+  return circumference.value;
+});
+
+const minutesLeft = computed(() => Math.floor(totalSecondsLeft.value / 60));
+
+const secondsLeft = computed(() => {
+  if (!props.isRunning) {
```

Comment:
what if timer is running at first, then isRunning is set to false, then we'll show wrong seconds and will show seconds from props, not seconds in minutes. We need to add a test for this case and let's rename this variable to minuteSecondsLeft

Reply:
the same computed is used also to show only seconds (line 54). i would not rename it

Reply:
You have to use totalSecondsLeft on line 54

Reply:
now it should be ok